2019/latex/ISMIR2019template.tex

```diff
@@ -0,0 +1,240 @@
+
+
+\subsection{Page Numbering, Headers and Footers}
+
+Do not include headers, footers or page numbers in your submission.
+These will be added when the publications are assembled.
+
+\section{First Level Headings}
+
+First level headings are in Times 10pt bold,
+centered with 1 line of space above the section head, and 1/2 space below it.
+For a section header immediately followed by a subsection header, the space should be merged.
+
+\subsection{Second Level Headings}
+
+Second level headings are in Times 10pt bold, flush left,
+with 1 line of space above the section head, and 1/2 space below it.
+The first letter of each significant word is capitalized.
+
+\subsubsection{Third and Further Level Headings}
+
+Third level headings are in Times 10pt italic, flush left,
+with 1/2 line of space above the section head, and 1/2 space below it.
+The first letter of each significant word is capitalized.
+
+Using more than three levels of headings is highly discouraged.
+
+\section{Footnotes and Figures}
+
+\subsection{Footnotes}
+
+Indicate footnotes with a number in the text.\footnote{This is a footnote.}
+Use 8pt type for footnotes. Place the footnotes at the bottom of the page on which they appear.
+Precede the footnote with a 0.5pt horizontal rule.
+
+\subsection{Figures, Tables and Captions}
+
+All artwork must be centered, neat, clean, and legible.
+All lines should be very dark for purposes of reproduction and art work should not be hand-drawn.
+The proceedings are not in color, and therefore all figures must make sense in black-and-white form.
+Figure and table numbers and captions always appear below the figure.
+Leave 1 line space between the figure or table and the caption.
+Each figure or table is numbered consecutively. Captions should be Times 10pt.
+Place tables/figures in text as close to the reference as possible.
+References to tables and figures should be capitalized, for example:
+see \figref{fig:example} and \tabref{tab:example}.
+Figures and tables may extend across both columns to a maximum width of 17.2cm.
+
+\begin{table}
+ \begin{center}
+ \begin{tabular}{|l|l|}
+  \hline
+  String value & Numeric value \\
+  \hline
+  Hello ISMIR  & \conferenceyear \\
+  \hline
+ \end{tabular}
+\end{center}
+ \caption{Table captions should be placed below the table.}
+ \label{tab:example}
+\end{table}
+
+\begin{figure}
+ \centerline{\framebox{
+ \includegraphics[width=\columnwidth]{figure.png}}}
+ \caption{Figure captions should be placed below the figure.}
+ \label{fig:example}
+\end{figure}
+
+\section{Equations}
+
+Equations should be placed on separate lines and numbered.
+The number should be on the right side, in parentheses, as in \eqnref{relativity}.
+
+\begin{equation}\label{relativity}
+E=mc^{2}
+\end{equation}
+
+\section{Citations}
```

Comment:
I wonder if we'd like to include scientific guidelines / reminders about e.g. citing late breaking demos, or other things discussed in  the community lately. Would this be an effective place to add this, do you think? To what extent is the paper template a submission contract, of sorts?

Reply:
Yeah, I like the idea of making the templates self-contained in terms of guidelines.

Reply:
I think this should be a markdown-ish thing on the conference website. Maybe we can also have on the ISMIR page. But good point though...